sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -1,20 +1,20 @@
 <template>
-  <v-alert :type="color" variant="outlined">
-    <template v-slot:title>
+  <v-alert :type="type" variant="outlined">
+    <template #title>
       <div class="label-bold-normal">{{ header }}</div>
     </template>
     <v-row>
-      <v-col cols="10"
+      <v-col sm="10" md="10" lg="10"
         ><div class="label-value-normal">{{ summary }}</div></v-col
       >
-      <v-col cols="2"><slot name="actions"></slot></v-col>
+      <v-col sm="2" md="2" lg="2"><slot name="actions"></slot></v-col>
     </v-row>
   </v-alert>
 </template>
 <script lang="ts">
 export default {
   props: {
-    color: {
+    type: {
```

Comment:
A validator can be added to the type. We have a couple of similar properties doing it already, like below:
```typescript
validator: (val: string) => val in LayoutTemplates
```